Dockerfile.deploy

```diff
@@ -0,0 +1,30 @@
+FROM centos:7
+LABEL maintainer "Devtools <devtools@redhat.com>"
+LABEL author "Devtools <devtools@redhat.com>"
+ENV LANG=en_US.utf8
+ENV F8_INSTALL_PREFIX=/usr/local/env
+
+# Create a non-root user and a group with the same name: "env"
+ENV F8_USER_NAME=env
+RUN useradd --no-create-home -s /bin/bash ${F8_USER_NAME}
+
+# Install little pcp pmcd server for metrics collection
+# would prefer only pmcd, and not the /bin/pm*tools etc.
+COPY pcp.repo /etc/yum.repos.d/pcp.repo
+RUN yum install -y pcp pcp-pmda-prometheus && yum clean all && \
+    mkdir -p /etc/pcp /var/run/pcp /var/lib/pcp /var/log/pcp  && \
+    chgrp -R root /etc/pcp /var/run/pcp /var/lib/pcp /var/log/pcp && \
+    chmod -R g+rwX /etc/pcp /var/run/pcp /var/lib/pcp /var/log/pcp
+COPY ./env+pmcd.sh /env+pmcd.sh
+EXPOSE 44321
+
+COPY bin/env ${F8_INSTALL_PREFIX}/bin/env
+COPY config.yaml ${F8_INSTALL_PREFIX}/etc/config.yaml
```

Comment:
I think that this should be used as a configmap, but I can be wrong

Reply:
Well.. this file is not used anywhere actually. The initial idea was that we can configure via a configuration file but it's not used.